### PR TITLE
Registered checkout

### PR DESCRIPTION
**Pull request by Eternal-Flame085**
- Finished user story 26
- changed the redirect in order create
- Created /profile/orders route and view page
- changed and added tests reflecting the features

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -31,6 +33,6 @@ def create
   private
 
   def order_params
-    params.permit(:name, :address, :city, :state, :zip)
+    params.permit(:name, :address, :city, :state, :zip, :status)
```
Comment (JBelta) [inline app/controllers/orders_controller.rb:36]: got a status. enum?

**Review by araspaas — APPROVED** [review]
Lookin good keep it up

**Comment by JBelta** [comment]
/)ºπº)/

**Comment by SageOfCode** [comment]
Looks ready to merge to me.

**Pull request merged**